Examples/test-suite/javascript/typescript_methods_runme.ts

```diff
@@ -0,0 +1,10 @@
+import { C } from './typescript-methods-types';
+
+const typescript_interface = require("./typescript_methods");
+
+const c: C = new typescript_interface.C();
+c.intValue = 3;
+c.floatValue = 2.45;
+const s = c.sum(2,3);
```

Comment:
Why not test the 2 other functions defined in `C`? As long as they're declared there, I think they should be tested.